cluster/addons/dns/kube2sky/kube2sky.go

```diff
@@ -531,7 +531,7 @@ func newKubeClient() (*kclient.Client, error) {
 	if masterURL != "" && *argKubecfgFile == "" {
 		// Only --kube-master-url was provided.
 		config = &restclient.Config{
-			Host:          masterURL,
```

Comment:
Hmm, I think kube2sky is deprecated and shouldn't be used anymore... Probably not worth to add effort into that